Sources/Plasma/Apps/plClient/plClient.cpp

```diff
@@ -1389,6 +1390,12 @@ bool plClient::StartInit()
     fConsole->RegisterAs( kConsoleObject_KEY );     // fixedKey from plFixedKey.h
     fConsole->Init( fConsoleEngine );
 
+#ifndef PLASMA_EXTERNAL_RELEASE
+    fRemoteConsole = new pfRemoteConsole();
+    fRemoteConsole->RegisterAs( kRemoteConsoleObject_KEY );
```

Comment:
It doesn't look like we ever need to address messages to the RemoteConsole by its key, so we probably don't need a fixed key entry for it.

Reply:
You still need a key to receive messages though, so this is not really incorrect.

Reply:
This is mostly copy pasted from the fConsole code above. I used to not have the `RegisterAs` call, but the messages wouldn't ever arrive to the console. Is there another way to register the `plRemoteConsole` without a fixed key?